src/plugins/crypto/README.md

```diff
@@ -109,62 +110,62 @@ Set the CMake variables `OPENSSL_INCLUDE_DIR` and `OPENSSL_LIBRARIES` to your de
 
 The crypto plugin will encrypt and decrypt values using AES-256 in CBC mode.
 
-The key derivation is still WIP.
+The GPG binary is required for handling the cryptographic keys.
 
-### Planned Features ###
+At the moment the plugins will only run on UNIX/Linux-like systems, that implement `fork ()`.
 
-- Encryption of values
-- Decryption of values
-- Key derivation by metadata (password provided within a meta-key)
-- Key derivation by using a specified key-file (like the SSH client does)
-- Key derivation by utilizing the pgp-agent
+## Examples ##
+
+To mount a backend with the gcrypt plugin variant that uses the GPG key 9CCC3B514E196C6308CCD230666260C14A525406, use:
 
-The encryption and decryption of values is a straight forward process, once the key and IV are supplied.
-The key-derivation process in a library is a bit tricky.
+	kdb mount test.ecf user/t crypto_gcrypt "crypto/key=9CCC3B514E196C6308CCD230666260C14A525406"
```

Comment:
When supplying no key at all you should receive the following error during the mount phase:

    Error (#130) occurred!
    Description: The configuration is invalid or incomplete.
    Ingroup: plugin
    Module: crypto
    At: /root/libelektra/src/plugins/crypto/gpg.c:219
    Reason: Missing GPG key (specified as /gpg/key) in plugin configuration.
    Mountpoint: 
    Configfile:

When supplying a wrong or missing key, you should get:

    Error (#151) occurred!
    Description: Error in the GPG module.
    Ingroup: plugin
    Module: crypto
    At: /root/libelektra/src/plugins/crypto/gpg.c:457
    Reason: GPG failed with return value 512. gpg: xyz: skipped: No public key
    gpg: [stdin]: encryption failed: No public key

The error message displayed by GPG is now appended to `errorKey` (since 9e18629ac2aeae664c224b6ef95b8ab9a9fa8abd )

In the first case we *could* display available keys. I had the same idea during development, so I will open an issue for the improvement.

When I entered a wrong key, I would not expect `kdb` to tell me which keys are available.